.github/workflows/pull_request.yml

```diff
@@ -172,7 +174,8 @@ jobs:
         with:
           name: test-reports
           path: test-reports
-  tvos_and_watchos:
+
+  tvos:
```

Comment:
Why are we splitting this up? Is this taking so much longer than `macos + ios` ?
Adding more `macos-latest` jobs will really slow down the build, since there is a low limit on how much we can run in parallel.

Reply:
yes, either that or I up the wait time

Reply:
Yes, let's combine them. Feel free to also move JS job into the Linux job at the same time.